common/util/LazyMergingStream.java

```diff
@@ -14,9 +14,10 @@
  *
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
  */
 
-package grakn.core.graql.util;
+package grakn.core.common.util;
```

Comment:
Make sure to split this class out of `//common` before we split `//common` into its own repository (https://github.com/graknlabs/grakn/issues/5454). We want to make sure that the `@graknlabls_common` repo only contain packages that are used by all other repos on @graknlabs.